_data/examples.yml

```diff
@@ -38,6 +38,18 @@
 
 
 
+# Tutorials hosted at https://github.com/oscar-system/notebooks
+
+- title: Elliptic Fibrations and Covered Schemes
+  repository: oscar-system/notebooks/
+  branch: master
+  filename: notebooks/elliptic_fibrations/elliptic_fibrations_and_covered_schemes
+  author: Simon Brandhorst, Matthias Zach
+  thumbnail: Schemes.png
+  language: julia
```

Comment:
Hello everyone! 

I am very sorry, but I realize only now that I must have sent the wrong link to @HereAround ! 

It should really be [this one here](https://github.com/HechtiDerLachs/OscarTutorials/blob/main/covered_schemes_and_elliptic_fibrations/elliptic_fibrations_and_covered_schemes.ipynb). That has also been accessible without password during the ICMS. 

My apologies again for all the effort you have probably all put into this already.